integration-cli/docker_cli_network_unix_test.go

```diff
@@ -585,13 +607,18 @@ func (s *DockerNetworkSuite) TestDockerNetworkConnectDisconnect(c *check.C) {
 	nr = getNwResource(c, "test")
 	c.Assert(nr.Name, checker.Equals, "test")
 	c.Assert(len(nr.Containers), checker.Equals, 0)
-
-	dockerCmd(c, "network", "rm", "test")
-	assertNwNotAvailable(c, "test")
 }
 
 func (s *DockerNetworkSuite) TestDockerNetworkIPAMMultipleNetworks(c *check.C) {
 	testRequires(c, SameHostDaemon)
+
+	defer func() {
+		for i := 1; i < 8; i++ {
```

Comment:
shouldn't we have a defer after each instead ? (and maybe an helper methods that group `dockerCmd` and `assertNwNotAvailable`)

Reply:
I don't quite understand, do you mean we add `defer` before each statement in the `for` loop like this:
```
for (...) {
      defer dockerCmd(...)
      defer assertNwNotAvailable(...)
  }
```
If that's your concern, IMO the `defer func() ` should handle that?
A helper method to group `dockerCmd` and `assertNwNotAvailable` is make sense, but can this be added later when we plan to migrate this test suite to other place?

Reply:
@arm64b nop I mean have the `defer` for remove network1 after the creation of network1, same for network2 and the rest.

Reply:
Ah, understood 😄 So for 2nd version, I will include the changes requested here and also add a new helper to consolidate the `dockerCmd` and `assertNwNotAvailable`, does this make sense for you?

Reply:
:+1:

Reply:
Will done in next week!